src/core/sock.h

```diff
@@ -165,6 +165,9 @@ int nn_sock_add_ep (struct nn_sock *self, struct nn_transport *transport,
 /*  Remove the endpoint with the specified ID from the socket. */
 int nn_sock_rm_ep (struct nn_sock *self, int eid);
 
+/* Return last_errno for the endpoint with the specified ID */
```

Comment:
nit. Add a period at the end of the sentence.  (To match other lines in this file.)